lib/src/components/NxTab/NxTab.scss

```
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-colors';
+ @import '../../scss-shared/nx-variables';
+
+.nx-tab {
+  border-bottom: 4px solid transparent;
+  cursor: pointer;
+  display: inline-block;
+  list-style: none;
+  padding: $nx-spacing-xs $nx-spacing-l;
+
+  &.active {
+    border-bottom: solid $nx-background-color-interactive;
```

This unsets the `4px` specified above, resulting in the browser-default 3px border width.  Given the above declaration, I'd say the most minimal and precise way you want to set the active style is this:
```suggestion
    border-bottom-color: $nx-background-color-interactive;
```